src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -23,7 +24,10 @@ specific language governing permissions and limitations under the License.
   box-shadow: 1px 0 0 0 var(--theme-tab-separator-color);
   color: var(--theme-tab-inactive-color);
   composes: app-no-drag from '../../../../../shared/widgets/helpers/common.css';
-  transition: width 0.05s var(--animation-curve), background-position 0.2s var(--animation-curve), height 0.2s var(--animation-curve);
+  transition:
+    width 0.05s var(--animation-curve),
+    height 0.2s var(--animation-curve),
+    background-position 0.2s var(--animation-curve);
```

Comment:
Nit: I'm noticing this now, but it looks like opacity isn't animated for opening tabs. If that's the case, let's remove the initial tab opacity rule from here: https://github.com/ericawright/tofino/blob/85e2e556ee8a7069f16993eb4dead9a33113098b/src/browser-frontend/views/browser/chrome/tabbar/tab.css#L48, along with the css variable if not needed anymore. Your choice if you want to do it as part of this PR or later (might as well do it now since we're touching all of this).